### PR TITLE
Depend on pocl with Isuru's OSX fixes

### PR DESCRIPTION
**Pull request by inducer**

### CODE REVIEW

**Comment by inducer** [comment]
cc @isuruf

**Comment by isuruf** [comment]
I can move `pocl=1.1_0` to a broken label since that version is clearly broken. (1/82 failed in Release and 22/82 failed in Debug. New version passes all tests in Release and Debug mode)

**Comment by inducer** [comment]
Yeah, I think that makes sense.

**Comment by inducer** [comment]
(And thanks for offering to do this!)

**Comment by isuruf** [comment]
Done. https://anaconda.org/conda-forge/pocl/files

**Comment by inducer** [comment]
Any clue what happened here:

```
The command "conda build ./recipe" exited with 1.
1.00s$ upload_or_check_non_existence ./recipe conda-forge --channel=main
Traceback (most recent call last):
  File "/Users/travis/miniconda3/bin/upload_or_check_non_existence", line 135, in <module>
    main()
  File "/Users/travis/miniconda3/bin/upload_or_check_non_existence", line 114, in main
    cli = get_binstar(argparse.Namespace(token=token, site=None))
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/binstar_client/utils/config.py", line 139, in get_binstar
    aserver_api = get_server_api(token, site, log_level, cls)
  File "/Users/travis/miniconda3/lib/python3.6/site-packages/binstar_client/utils/config.py", line 120, in get_server_api
    return cls(token, domain=url, verify=verify, **kwargs)
TypeError: 'int' object is not callable
The command "upload_or_check_non_existence ./recipe conda-forge --channel=main" exited with 1.
```
?

**Comment by isuruf** [comment]
New `anaconda-client` version that breaks compatibility.

**Comment by inducer** [comment]
So, rerender?

**Comment by inducer** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -42,7 +42,7 @@ requirements:
         - mingwpy  # [win and not py3k]
 {% if osx_variant == "pocl" %}
         - osx-pocl-opencl
-        - pocl >=1.1
+        - pocl >=1.1-1
```
Comment (isuruf) [inline recipe/meta.yaml]: I think this should be `1.1_1`

**Comment by inducer** [comment]
Fixed, thanks!

**Comment by inducer** [comment]
```
Can't build /Users/travis/build/conda-forge/pyopencl-feedstock/recipe due to unsatisfiable dependencies:
  - pocl >=1.1_1
```

Any clues? Seems fine to me.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -42,7 +42,7 @@ requirements:
         - mingwpy  # [win and not py3k]
 {% if osx_variant == "pocl" %}
         - osx-pocl-opencl
-        - pocl >=1.1
+        - pocl >=1.1_1
```
Comment (isuruf) [inline recipe/meta.yaml]: It seems specifying the build number in inequalities is not supported

Comment (isuruf) [reply recipe/meta.yaml]: `!=1.1 0` works, but `>=1.1,!=1.1 0` doesn't.

Comment (inducer) [reply recipe/meta.yaml]: OK. Thanks for following up. I'll just keep it simple, with `>=1.1`, and rely on the broken tag to keep `1.1_0` out.

**Comment by inducer** [comment]
Thanks!

**Pull request merged**